docs/blog/2018-10-16-unstructured-data/index.md

```diff
@@ -0,0 +1,206 @@
+---
+title: Using unstructured data in Gatsby
+date: 2018-10-16
+author: Amberley Romo
+tags: ["sourcing", "data"]
+---
+
+Most examples in the Gatsby docs and on the web at large focus on leveraging source plugins to manage your data in Gatsby sites. And rightfully so! Gatsby's data layer is powerful and extremely effective.
```

Comment:
I'd start with a more general introduction to the problem of sourcing & transforming data into a website before this paragraph to set the tone. It feels like it's missing the introduction currently.

Reply:
Lmk what you think about the additions